packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -824,23 +818,29 @@ export const Table = <
         data-iui-size={density === 'default' ? undefined : density}
         {...ariaDataAttributes}
       >
-        <div
-          className='iui-table-header-wrapper'
-          ref={headerRef}
-          onScroll={() => {
-            if (headerRef.current && bodyRef.current) {
-              bodyRef.current.scrollLeft = headerRef.current.scrollLeft;
-              updateStickyState();
-            }
-          }}
-        >
-          <div className='iui-table-header'>
-            {headerGroups.slice(1).map((headerGroup: HeaderGroup<T>) => {
-              const headerGroupProps = headerGroup.getHeaderGroupProps({
-                className: 'iui-table-row',
-              });
-              return (
-                <div {...headerGroupProps} key={headerGroupProps.key}>
+        {headerGroups.map((headerGroup: HeaderGroup<T>) => {
+          headerGroup.headers = headerGroup.headers.filter(
```

Comment:
This filter removes unnecessary headers that are being placed when the `Header` is specified for tables with multi-select or expandable rows. This shouldn't be the permanent solution because users who put their header ids as either of these strings will not have their header shown

![TableRowHeader](https://user-images.githubusercontent.com/78564221/200916319-72b3b823-c567-4c7b-a176-9b49d5d0e811.gif)

Reply:
Maybe make a comment about needing a better solution here so we won't forget, or make another issue?

Reply:
Comment has been added